frontend/src/scenes/data-warehouse/editor/sidebar/QueryInfo.tsx

```diff
@@ -164,6 +178,80 @@ export function QueryInfo({ codeEditorKey }: QueryInfoProps): JSX.Element {
                         )}
                     </div>
                 </div>
+                {savedQuery && (
+                    <>
+                        <div>
+                            <h3>Materialization Runs</h3>
+                            <p>The last runs for this materialized view. These can be scheduled or run on demand.</p>
+                        </div>
+                        <LemonTable
+                            loading={initialDataWarehouseSavedQueryLoading}
+                            dataSource={dataModelingJobs?.results || []}
+                            columns={[
+                                {
+                                    title: 'Status',
+                                    dataIndex: 'status',
+                                    render: (_, { status, error }: DataModelingJob) => {
+                                        const type =
+                                            status === 'Completed'
+                                                ? 'success'
+                                                : status === 'Failed'
+                                                ? 'danger'
+                                                : 'warning'
```

Comment:
I _despise_ nested ternaries; they're so hard to read. Could we just have a mapping of `status` -> `type` and just index into it instead?